app/views/reviews/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1> Create a review </h1>
+
+  <%= form_tag "/shelters/#{@shelter.id}/new_review" do %>
+    <%= label_tag :title %>
+    <%= text_field_tag :title %></br>
+
+    <%= label_tag :rating %>
+    <%= select_tag :rating, options_for_select([1,2,3,4,5]) %></br>
```

Comment:
Super job with the rating system. I wasn't sure how to handle this, and you rocked it!